applications/GeoMechanicsApplication/custom_constitutive/linear_elastic_law.cpp

```diff
@@ -89,7 +87,6 @@ double& GeoLinearElasticLaw::CalculateValue(ConstitutiveLaw::Parameters& rParame
     if (rThisVariable == STRAIN_ENERGY) {
         Vector& r_strain_vector = rParameterValues.GetStrainVector();
         Vector& r_stress_vector = rParameterValues.GetStressVector();
-        this->CalculateCauchyGreenStrain(rParameterValues, r_strain_vector);
```

Comment:
Removing this line means that now the strain should be provided when calling this function. From the geo side, I don't see any usages of this function with STRAIN_ENERGY as input, but it's there for some structural elements. However, since these have their own laws, I don't see this as a big problem.